src/modules/auth/components/customdata.scss

```diff
@@ -19,7 +19,7 @@
 .custom-data-card {
 	grid-area: q;
     .card-body-scroll-sm {
-        height: 200px !important;
+        height: 150px !important;
```

Comment:
@petrKavulok Are you sure about it? Why did you make the card body scroll height smaller?

Reply:
because otherwise, the custom data card is 50px bigger, which causes the 'Uživatelské údaje' and 'Poslední přihlášení' to also increase in size. And since we are not expecting any more entries in these two cards, I decreased the height of custom data. (it was Aleš' idea)